grayskull/strategy/py_base.py

```diff
@@ -602,6 +603,13 @@ def get_test_imports(metadata: dict, default: Optional[str] = None) -> List:
     return result
 
 
+def get_test_requirements(metadata: Dict) -> List:
+    try:
+        return metadata["extras_require"][CLIConfig().extras_require_test]
```

Comment:
Just to cover a few corner case we might change it to
```suggestion
        return metadata["extras_require"][geattr(CLIConfig(), "extras_require_test", [])]
```